Description: Use /usr/share/common-licenses instead of user's home directory
 in tests.
Author: Andrius Merkys <merkys@debian.org>
--- a/t/File-DirList.t
+++ b/t/File-DirList.t
@@ -13,8 +13,8 @@
 # Insert your test code below, the Test::More module is use()ed here so read
 # its man page ( perldoc Test::More ) for help writing this test script.
 
-my $homeDir = ( "\L$^O" =~ m/win32/ ) ? '.' : glob('~');
-print STDERR ("\nListing your home dir '$homeDir' for test purposes\n");
+my $homeDir = '/usr/share/common-licenses';
+print STDERR ("\nListing common-licenses dir '$homeDir' for test purposes\n");
 foreach my $item (@{File::DirList::list($homeDir, 'din', "\L$^O" =~ m/win32/ ? 1 : 0, 1, 1)})
 	{ print STDERR (sprintf("%s'%s'%s\n", ($item->[14] ? 'dir  ' : 'file '), $item->[13], (!$item->[15] ? '' : ($item->[15] < 0 ? ' bad' : '').' link to \''.$item->[16].'\''))); };
 ok("listing completed");
